packages/core/phase2/createPncDisposal.ts

```diff
@@ -26,7 +26,7 @@ const createPncDisposal = (
   const qtyDuration = durationUnit ? durationUnit + durationLength?.toString() : ""
   return {
     qtyDuration: qtyDuration,
-    qtyMonetaryValue: amtSpecInResult ? amtSpecInResult : "",
+    qtyMonetaryValue: amtSpecInResult?.toString(),
```

Comment:
`qtyMonetaryValue` is a string type in the schema, but we assigned a number value here and force cast to `PncDisposal` to bypass ts error.